src/screens/Followings/FollowersScreen.js

```diff
@@ -88,6 +95,10 @@
       unsubFocusListener();
     };
   }, []);
+
+  const followedUsers = dataFollower.filter((item) => item.user.following);
+  const unfollowedUsers = dataFollower.filter((item) => !item.user.following);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The separation of followed and unfollowed users into two different arrays (`followedUsers` and `unfollowedUsers`) is a good practice for better data management. However, ensure that `item.user.following` always returns a boolean value to prevent unexpected results.
```diff
+ const followedUsers = dataFollower.filter((item) => item.user.following);
+ const unfollowedUsers = dataFollower.filter((item) => !item.user.following);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->